### PR TITLE
CR-899 Added support for Household-Information page

### PR DESCRIPTION
**Pull request by PeterBochel**
Updating Locust script to support the latest RHUI:

- Following 'request a new access code' link, other enter-postcode page fails.
- Now expects and handles the Household-Information page.

Update 4th Dec:
Now also supports fulfilment case for a postcode with more than 100 addresses (would previously fail to find the postcode and fail with obscure secondary errors).

### CODE REVIEW

File: locust_tasks/locustfile.py [hunk 1]
```diff
@@ -311,7 +367,17 @@ def confirm_address(self):
...
+
+    @task
+    def request_new_household_access_code(self):
+        """
+        POST 'Continue' to confirm the request of a new household access code
+        """
+        with self.client.post("/en/requests/access-code/household-information/", {
+           'form-confirm-address': 'yes' 
+        }, catch_response=True) as response:
+            verify_response('RequestUacSms-6-Household', self, response, 200, Page.SELECT_METHOD)
```
Comment (rpcatling) [inline locust_tasks/locustfile.py]: The identity probably wanted to be 'RequestUacPost-6-Household' here. copy and paste error from the SMS stuff ?

**Review by rpcatling — CHANGES_REQUESTED** [review]
Only one actually (minor) error spotted.
There are lots of code-style issues (if you open it in pycharm), but consider fixing these on the next iteration:
- initialising all self variables in the class `__init__` method  before use (consider an intermediate class of SequentialTaskSet to do this.
- remove TaskSet import (not used)
- some variables shadowing higher level variables (eg id and task) . consider renaming these to identity and taskset
- some long lines could be shortened.
- move all method and class comments below the method / class signature
- method `extractAddressRadioButtonValue` is not a pythonic method name

Let pycharm be your guide if you do the tidy up exercise.

File: locust_tasks/locustfile.py (hunk 1 above)
Comment (PeterBochel) [reply locust_tasks/locustfile.py]: Typo fixed.
Have moved initialisation code into an on_start method.
Can't get too excited about pycharm (perf test lifetime too short to pay back the investment)

**Review by rpcatling — APPROVED** [review]

File: locust_tasks/locustfile.py (hunk 1 above)
Comment (rpcatling) [reply locust_tasks/locustfile.py]: Not sure I agree with you about pycharm, but won't stop approval.

**Review by cookel2 — APPROVED** [review]
Looks good to me

**Pull request merged**